4-esqueleto/README 12.11.39.md

```diff
@@ -0,0 +1,140 @@
+
+![nuevacontra](/4-esqueleto/nuevacontra.png)
+
+Una vez que el usuario inicie sesión y por lo tanto, acceda a su cuenta personal encontrará a su disposición el sitio web con el contenido y tambien un apartado llamado "Adjunta tus propuestas". 
+
+
+#### *Wireframe* de "LOG IN" 
+
+![loginowire](/4-esqueleto/loginwire.png)
+
+Una vez introducida correctamente la contraseña, sin problemas el usuario podrá acceder a la página web donde encontrará lo siguiente:
+
+![REGISTRADOS](/4-esqueleto/portadaregistrados.png)
+
+
+#### 3.1. Ajunta tus propuestas
+En el apartado "Adjunta tus propuestas" el usuario registrado tiene la opción de adjuntar los archivos de creación propia relacionados con la igualdad de género. 
+
+![adjunta](/4-esqueleto/adjunta.png)
+
+Una vez adjuntado el archivo deseado podrás ver todos los archivos que has subido en la siguiente imagen. 
+
+![tuspropuestas](/4-esqueleto/tuspropuestas.png)
+
+El tic verde indica que el administrador del sitio web ha revisado los archivos y que le ha dado el visto bueno para poder publicarlo. La interrogación significa que el archivo todavía está en la fase de revisión. Y finalmente, la cruz roja significa que el archivo no se va a publicar. 
+
+![tuspublicaciones](/4-esqueleto/tuspublicaciones.png)
+
+En el apartado “Tus publicaciones” se encuentran aquellos archivos que han sido revisados y aceptados por el administrador. En la parte derecha de cada archivo aparece la opción “Acceder” que permite acceder a la publicación en cuestión ya publicada a disposición de cualquier usuario. 
+
+En este apartado, nuestra página contará con un elemento de localización, que permitirá a cada usuario saber dónde se encuentra en cada momento durante su navegación, pudiendo volver hacia atrás a través de esta ruta con hipervínculos.
+
+Además, la página también contará con un Mapa del Sitio. Con él, permitiremos al usuario en cuestión saltar a la página que desee.
+
+
+## DISEÑO DE LA NAVEGACIÓN 
+
+### Organización de ruta
+En este apartado, nuestra página contará con un elemento de localización, que permitirá a cada usuario saber dónde se encuentra en cada momento durante su navegación, pudiendo volver hacia atrás a través de esta ruta con hipervínculos.
+
+Además, la página también contará con un Mapa del Sitio. Con él, permitiremos al usuario en cuestión saltar a la página que desee.
+
+### Navegación 
+Así mismo, también hemos decidido que nuestra página sea de navegación a través de barras y pestañas. Concretamente, hemos decidido que lleve un menú dinámico dentro de ella. 
+
+Dentro de cada temática existirán subtemas que podrán ser seleccionados a través de pop up menus. Esto hará que nuestra web posea un aspecto moderno y sencillo de llevar.
+
+
+## DISEÑO DE LA INFORMACIÓN 
```

Comment:
Aquí falta definir muchos elementos del sitio:

- ¿cómo se llama cada categoría?
- ¿cómo os referís al proceso de acceso/login, registro, adjuntar propuestas...?
 - ...

Las palabras concretas que usáis para todos estos elementos son parte del diseño de la información.